@types/index.d.ts

```diff
@@ -61,7 +61,7 @@ declare module "opentok-react-native" {
     stream: Stream;
   }
 
-  interface OTSessionViewProps extends ViewProps {
+  interface OTSessionProps extends ViewProps {
```

Comment:
Can you explain to me why you are renaming the interfaces without the `View` keyword in the name please?

Reply:
IMO it's confusing to have `OTSubscribeViewProps`, it belongs to `OTSubscribeView` or `OTSubscribe`? And it's a common practice to add only the Props to the name of the components to type your Props.

An example from Material UI: https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Accordion/Accordion.d.ts

So I thought it was better to have a direct name related to the component. If the component name is `OTSubscribe` the Props type should be `OTSubscribeProps`.

Reply:
That's a good point, thanks for explaining. I will bring it internally and move forward with the PR 😃

Reply:
@enricop89 Any update about it?